beacon_node/src/config.rs

```diff
@@ -382,10 +384,30 @@ pub fn get_config<E: EthSpec>(
         client_config.execution_layer = Some(el_config);
     }
 
+    // 4844 params
+    client_config.trusted_setup = context
+        .eth2_network_config
+        .as_ref()
+        .and_then(|config| config.kzg_trusted_setup.clone());
```

Comment:
Is it possible / worth avoiding the clone of `TrustedSetup` here, given the trusted setup is quite large for mainnet?

Reply:
I tried to do this for a bit, but it ends up being not easy because the network config here is behind an `Arc`, and the trusted setup type and the methods using it are being defined in the c-kzg lib, so it's not as simple as also `Arc` ing that (I went down that rabbit hole). I think in the future the easiest route route be to improve the c-kzg api so we can `Arc` it in our codebase